_data/contents.yml

```diff
@@ -16,6 +16,7 @@ typeclasses:
   - enumeration
   - string-representation
   - monoids
-  - iteration
+  - constructor-classes
   - functors
```

Comment:
Yeah, that's right. I had previously used "iteration" as the tag when the lesson was going to lead up to Traversable, since the whole thing was going to build up to what has been described as ["essence of the iterator pattern"](https://www.cs.ox.ac.uk/jeremy.gibbons/publications/iterator.pdf), but since dropping Traversable from the lesson I renamed it from "iteration" to "constructor-classes".

Reply:
I realise now that the GitHub UI isn't the best when I thought I had highlighted one particular line. The `functor` lesson is also absorbed, right?

Reply:
Oh, I see. The one called "functors" is Applicative/Monad.